website/util/__init__.py

```diff
@@ -7,9 +7,9 @@
 import furl
 
 from flask import request, url_for
-from django.core.urlresolvers import reverse
 
-from website import settings
+from website import settings as osf_settings
+from api.base import settings as drf_settings
```

Comment:
Maybe api_settings; this might not be the last django thing we work in like this.